mesop/server/config.py

```diff
@@ -0,0 +1,29 @@
+import os
+from typing import Literal
+
+from dotenv import load_dotenv
+from pydantic import BaseModel
+
+load_dotenv()
+
+
+class Config(BaseModel):
+  """Mesop app configuration."""
+
+  secret_key: str = ""
+  state_session_backend: Literal["memory", "null"] = "memory"
```

Comment:
Yes, the issue I ran into here is that if I did None, the user would need to specify an environment variable, otherwise if not set the default would be none. And if I set a default if the environment variable was not set, then None would not be an option.

I guess in this case I could make it so empty string gets converted to None.

In the end, I renamed "null" to "none". But still need to explicitly specify "none"